contracts/FungibleTokenSwitchboard.cdc

```diff
@@ -0,0 +1,151 @@
+import FungibleToken from "./FungibleToken.cdc"
+
+/// FungibleTokenSwitchboard
+///
+/// The contract that allows an account to receive payments in multiple fungible
+/// tokens using a single {FungibleToken.Receiver}
+///
+pub contract FungibleTokenSwitchboard {
+    
+    // Storage and Public Paths
+    pub let SwitchboardStoragePath: StoragePath
+    pub let SwitchboardPublicPath: PublicPath
+    pub let SwitchboardReceiverPublicPath: PublicPath
+    
+    /// FungibleTokenSwitchboardInitialized
+    ///
+    /// The event emitted when the contract is created
+    ///
+    pub event FungibleTokenSwitchboardInitialized()
```

Comment:
probably don't need this

Reply:
There is no real need of emitting an event every time a contract is deployed right?

Reply:
Yeah, there is no need for that. Sometimes it can be useful as well but generally, it depends on how systems are designed, Ex - when you have an off-chain service that is monitoring the actions of a particular address and once it sees that initialized event so it can start watching the activities of the newly deployed contracts as well. 

When seeing that event it can initialize the tables and whatnot within the off-chain database to curate the activity for the newly deployed contract.